.github/workflows/phpunit-tests.yml

```diff
@@ -0,0 +1,144 @@
+name: PHPUnit Tests
+
+# Since Unit Tests are required to pass for each PR,
+# we cannot disable them for documentation-only changes.
+on:
+    pull_request:
+    push:
+        branches:
+            - trunk
+            - 'release/**'
+            - 'wp/**'
```

Comment:
Same remark about adding the `workflow_dispatch`

Reply:
Fixed in https://github.com/WordPress/gutenberg/commit/d448a66025a438b8a318c1711159f98043053c16